account_customer_wallet/models/res_company.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Coop IT Easy SCRL fs
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class Company(models.Model):
+    _inherit = "res.company"
+
+    customer_wallet = fields.Boolean(
```

Comment:
I don't remember when I commented that point, but I think this field is unnecessary. we could just remove it and assume that if ``customer_wallet_account_id`` is defined, the feature is active. So it will simplify a lot the ``res.config.settings`` (no need to onchange and raise error). 
the ``is_enabled_customer_wallet`` of ``pos.config`` could became a computed non stored field, based on the ``company_id.customer_wallet_account_id`` field)

What do you think ?

Reply:
@carmenbianca I agree with @legalsylvain

Reply:
:thumbsup: I remove some code then. I love removing code :D

Reply:
;-)

Reply:
[7974b1f](https://github.com/coopiteasy/addons/pull/222/commits/7974b1fd773e670140e6b639a60958baf6d379ae) code removed